project.godot

```diff
@@ -442,6 +442,10 @@ file_logging/max_log_files=8
 
 limits/message_queue/max_size_kb=8192
 
+[mono]
+
+project/assembly_name="Thrive"
```

Comment:
I think I've seen this change happen a lot. I guess it is finally time to allow this, I'll assume that this does not break anything as this seems like just the default value is now added.